guides/SPI.md

```diff
@@ -21,6 +21,21 @@ In order to become a _service provider_, you have two options:
 * Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+## Integration with Wix
+In order to enable Wix applications to communicate with your provider application, Wix needs to know some configuration details. You add the configurations via the Dev Center by entering your app’s url and method and other configuration details in the **JSON Editor** in the service’s extension description page. 
+The **Instructions** section of the extension description page provides a list of the required configuration parameters and types. 
+
+![alt text](../media/spi-config.png)
+
+
+In order to integrate the app you registered in the Wix Developer Center, provide configuration details in the **JSON Editor**:
+1. Click on **Components** in the Dashboard sidebar.
+2. Click on the **Add Component** button to drop down a list of component types.
+3. Select the **Integration component** option.
```

Comment:
Combine these:

```suggestion
2. Select **Add Component** > **Integration component**.
```

Alternatively, if you want to give a hint as to where to find this, you can precede with something like "In the upper right corner of the page, select **Add Component**..."